tests/unit/test_prompt_manager.py

```diff
@@ -310,3 +310,80 @@ def test_prompt_manager_custom_system_prompt_filename_not_found(prompt_dir):
         match=r'System prompt file "non_existent\.j2" not found at .*/non_existent\.j2\. Please ensure the file exists in the prompt directory:',
     ):
         PromptManager(prompt_dir=prompt_dir, system_prompt_filename='non_existent.j2')
+
+
+def test_jinja2_template_inheritance(prompt_dir):
```

Comment:
@OpenHands In tests/unit/test_prompt_manager.py -- PLEASE modify test_jinja2_template_inheritance so that we don't define anyblock in system_prompt.j2, and only import system prompt.j2 on other files